app/assets/stylesheets/govuk-component/_govspeak-html-publication-print.scss

```diff
@@ -1,5 +1,3 @@
-.govuk-govspeak-html-publication {
-  .govuk-sticky-element {
-    display: none;
-  }
+.govuk-sticky-element {
```

Comment:
Yeah, problem is that the .govuk-govspeak-html-publication class doesn't exist anymore (I think it must have been removed during the visual rework recently).

Feel like this is turning into a wider issue so I'm going to remove this change entirely and create a new card to look at it separately.